network/debinterfaces/bridge.go

```diff
@@ -19,6 +19,8 @@ var bridgeOnlyOptions = []string{
 	"dns-nameservers",
 	"dns-search",
 	"dns-sortlist",
+	"post-up",
+	"pre-down",
```

Comment:
This catches all pre/post-up/down instructions. As far as I know, MAAS currently uses pre/post-up/down only for routes, so this won't create any problems now.

However, this might change in the future. It might start doing something with the interface itself. For example enable/disable some hardware functions on the nic. In that case those pre/post-up/down should stay where they are and not get moved to bridge.